client/components/PetByTypeTile.js

```diff
@@ -1,6 +1,7 @@
 import React from "react"
+import {Link} from "react-router-dom"
 
-const PetByTypeTile = ({ petType: { name, age, imgUrl, vaccinationStatus } }) => {
+const PetByTypeTile = ({ petType: { id, name, age, imgUrl, vaccinationStatus } }) => {
   let ifVaccinated = "No"
 
```

Comment:
This is great! A possible refactor is to use a ternary operator during assignment.
```
const isVaccinated = (vaccinationStatus) ? "Yes" : "No"
```